monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js

```diff
@@ -12,12 +12,12 @@ export function renderMachineFromSystemData(data) {
     let machineStr = data['hostname'] + " ( ";
     data['ips'].forEach(function(ipInfo){
       if(typeof ipInfo === "object"){
-        machineStr += ipInfo['addr'] + " ";
+        machineStr += ipInfo['addr'] + ", ";
       } else {
-         machineStr += ipInfo + " ";
+         machineStr += ipInfo + ", ";
       }
     });
-    return machineStr + ")"
+    return machineStr.slice(0, -2) + " )"
```

Comment:
These changes add "," to machine's representation: `machine ( 192.168.56.1 127.0.0.1)` vs `machine ( 192.168.56.1, 127.0.0.1)`

Reply:
1. If you thought that was unclear, it's better to use a comment. `# This removes the last comma and space`
2. you can use `machineStr[0:-2]`

Reply:
2. In javascript?